tests/x11/gnomecase/application_starts_on_login.pm

```diff
@@ -18,38 +18,36 @@ use version_utils qw(is_leap is_sle is_tumbleweed);
 
 sub tweak_startupapp_menu {
     my ($self) = @_;
-    unless (is_sle('15+')) {
+    if (is_tumbleweed) {
+        x11_start_program 'gnome-tweaks';
+    }
+    elsif (is_sle('15+')) {
+        # tweak-tool entry is not in gnome-control-center of SLE15;
+        x11_start_program 'gnome-tweak-tool';
+    }
+    else {
         $self->start_gnome_settings;
         type_string "tweak";
         assert_screen "settings-tweak-selected";
         send_key "ret";
     }
-    else {
-        # tweak-tool entry is not in gnome-control-center of SLE15;
-        if (is_tumbleweed) {
-            x11_start_program 'gnome-tweaks';
-        }
-        else {
-            x11_start_program 'gnome-tweak-tool';
-        }
-    }
     assert_screen "tweak-tool";
     # increase the default timeout - the switching can be slow
     send_key_until_needlematch "tweak-startapp", "down", 10, 2;
 }
 
 sub start_dconf {
     my ($self) = @_;
-    unless (is_sle('15+')) {
+    if (is_tumbleweed && !is_sle('15+')) {
```

Comment:
One condition should be removed - statement seems duplicit.
If it `is_tumbleweed` then it can't be `is_sle` automatically, no need to check it..

Reply:
Indeed! It actually should be `||` instead. I'll fix it now